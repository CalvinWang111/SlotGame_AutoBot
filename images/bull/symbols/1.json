{
    "info_path": "images\\bull\\info\\1.png",
    "info_shape": {
        "width": 1080,
        "height": 1920
    },
    "symbols": [
        {
            "name": "wild",
            "path": "images\\bull\\symbols\\base_game\\wild.png",
            "bbox": [
                138,
                800,
                175,
                141
            ]
        },
        {
            "name": "big_treasure",
            "path": "images\\bull\\symbols\\base_game\\big_treasure.png",
            "bbox": [
                163,
                1200,
                124,
                333
            ]
        }
    ]
}