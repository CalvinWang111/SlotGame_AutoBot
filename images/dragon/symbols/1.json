{
    "info_path": "images\\dragon\\info\\1.png",
    "info_shape": {
        "width": 1920,
        "height": 1080
    },
    "symbols": [
        {
            "name": "wild",
            "path": "images\\dragon\\symbols\\test_game\\wild.png",
            "bbox": [
                397,
                182,
                275,
                333
            ]
        },
        {
            "name": "free_game",
            "path": "images\\dragon\\symbols\\test_game\\free_game.png",
            "bbox": [
                405,
                619,
                265,
                245
            ]
        }
    ]
}