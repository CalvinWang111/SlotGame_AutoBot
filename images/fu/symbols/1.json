{
    "info_path": "images\\fu\\info\\1.png",
    "info_shape": {
        "width": 1920,
        "height": 1080
    },
    "symbols": [
        {
            "name": "wild",
            "path": "images\\fu\\symbols\\test_game\\wild.png",
            "bbox": [
                365,
                203,
                222,
                182
            ]
        },
        {
            "name": "green_fu",
            "path": "images\\fu\\symbols\\test_game\\green_fu.png",
            "bbox": [
                354,
                446,
                245,
                203
            ]
        },
        {
            "name": "red_fu",
            "path": "images\\fu\\symbols\\test_game\\red_fu.png",
            "bbox": [
                359,
                695,
                233,
                206
            ]
        }
    ]
}