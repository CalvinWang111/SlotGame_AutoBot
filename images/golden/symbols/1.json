{
    "info_path": "images\\golden\\info\\1.png",
    "info_shape": {
        "width": 1080,
        "height": 1920
    },
    "symbols": [
        {
            "name": "wild",
            "path": "images\\golden\\symbols\\test_game\\wild.png",
            "bbox": [
                141,
                404,
                194,
                161
            ]
        },
        {
            "name": "coin",
            "path": "images\\golden\\symbols\\test_game\\coin.png",
            "bbox": [
                142,
                636,
                192,
                158
            ]
        },
        {
            "name": "jackpot",
            "path": "images\\golden\\symbols\\test_game\\jackpot.png",
            "bbox": [
                146,
                856,
                192,
                162
            ]
        },
        {
            "name": "coin_four_way_spread",
            "path": "images\\golden\\symbols\\test_game\\coin_four_way_spread.png",
            "bbox": [
                144,
                1084,
                196,
                160
            ]
        },
        {
            "name": "coin_eight_way_spread",
            "path": "images\\golden\\symbols\\test_game\\coin_eight_way_spread.png",
            "bbox": [
                144,
                1304,
                196,
                162
            ]
        }
    ]
}